app/Http/Controllers/UserController.php

```diff
@@ -63,5 +63,28 @@ public function loginUser(Request $request)
         return response()->json(['user' => $user]);
     }
 
+    public function logout(Request $request)
+{
+    $request->user()->currentAccessToken()->delete();
```

Comment:
in the user services do the logout logic, than in the controller you only call the service